arches_lingo/src/arches_lingo/components/scheme/report/SchemeNamespace.vue

```diff
@@ -107,6 +114,10 @@ function onNamespaceNameUpdate(val: string) {
                 :mode="EDIT"
                 @update="onNamespaceNameUpdate"
             />
+            <Button
```

Comment:
updating the Namespace && refreshing the page causes it to be loaded with an `[object Object]`

<img width="1193" alt="Screenshot 2025-01-02 at 11 21 44 AM" src="https://github.com/user-attachments/assets/c2054e62-212b-46ab-896d-850246e67838" />

Reply:
I don't observe this.  Is your graph up to date?
![Screenshot 2025-01-02 at 12 46 07 PM](https://github.com/user-attachments/assets/039d9260-34c4-40b5-81c7-02316ce9f261)

Reply:
oh fair yeah 👍